src/token/index.ts

```diff
@@ -3,8 +3,7 @@ export { TokenChip } from './components/TokenChip';
 export { TokenImage } from './components/TokenImage';
 export { TokenRow } from './components/TokenRow';
 export { TokenSearch } from './components/TokenSearch';
-export { TokenSelector } from './components/TokenSelector';
-export { TokenSelectorDropdown } from './components/TokenSelectorDropdown';
+export { TokenSelectDropdown } from './components/TokenSelectDropdown';
```

Comment:
Let's hold off on exporting `TokenSelectButton`. Don't think it's too useful for users.

Reply:
Yeah I can see that. Good call